### ISSUE TITLE
Crashes on startup.

### ISSUE THREAD

**Issue by AnarchyBlues** [heatray/WUMPatch#6]
GOG version, build 1077. It crashes as soon as the spinny logo thing shows up.

**Comment by heatray** [comment]
can you provide XOM*.log ?

**Comment by AnarchyBlues** [comment]
[XOM0-DESKTOP-IM0UK17.log](https://github.com/heatray/WUMPatch/files/12893953/XOM0-DESKTOP-IM0UK17.log)

Sorry for the late reply, I assume you mean this?

**Comment by heatray** [comment]
I can assume that the problem is with the shaders
```
   4747 | * XSOgl      | Invalid parameter type for rcpFrame in program CopyFxaa
   4770 | * XSOgl      | Invalid parameter type for rcpFrame in program CopyFxaaSepia
```

but in any case, restore the original files and try to install the patch in portions
- **dinput8.dll** & **plugins** - exe hacks
- **Data/Bundles** - aimers and weapon panel fix
- **Data/Tweak/MENUTWKXPCCOMMON.XOM** - menu fix
- **Data/Language** & **Data/Tweak/MENUTWKXPCEURO.XOM** - additional languages stuff
- **CG** - shaders

**Comment by AnarchyBlues** [comment]
Yeah so, it looks like it's the ASI loader itself that's crashing the game, I placed just dinpu8.dll in the game folder and it crashed.

That being said, I think you can close this as solved because I had a similar issue with GTA4 a while back, also caused by Ultimate ASI Loader. I don't know why it crashes when I use it, I have all the dependencies installed and the other ASI loaders worked fine with that game. Is there an alternative you can recommend for this mod?

**Comment by AnarchyBlues** [comment]
If I run the game with everything but the dinput8.dll and plugins I get this, so I'm guessing that is essential:
![image](https://github.com/heatray/WUMPatch/assets/98670485/c8abc4a6-96fc-4989-a39b-07680cef3f8f)

**Comment by heatray** [comment]
is vcredist installed? https://aka.ms/vs/17/release/vc_redist.x86.exe

**Comment by AnarchyBlues** [comment]
Yes.

**Comment by AnarchyBlues** [comment]
Ran a repair but it didn't change anything.

**Comment by heatray** [comment]
try new version https://github.com/heatray/WUMPatch/releases/tag/v23.11.12

**Comment by AnarchyBlues** [comment]
Tried it on a clean install, doesn't look like anything's changed. Is there a different ASI loader I can try?
[XOM0-DESKTOP-LI6Q9KV.log](https://github.com/heatray/WUMPatch/files/13336411/XOM0-DESKTOP-LI6Q9KV.log)

**Comment by heatray** [comment]
you can try any other version of [asi loader](https://github.com/ThirteenAG/Ultimate-ASI-Loader/releases)
also you can try inject patch using [SpecialK](https://github.com/SpecialKO/SpecialK) or [Koaloader](https://github.com/acidicoala/Koaloader) (I haven't tried)

**Comment by AnarchyBlues** [comment]
Tried them, no luck. Game crashes just by having the ASI loader in the game folder, doesn't even need the mod.

**Comment by AnarchyBlues** [comment]
I think I'll try posting this on the ASI Loader hub, since this doesn't seem to be a mod issue.

**Comment by AnarchyBlues** [comment]
Unfortunately I have not found a fix for this yet.

**Comment by AnarchyBlues** [comment]
Fixed, apparently a lot of old games don't work with ASI loader if Galaxy Overlay is enabled.

**Comment by AnarchyBlues** [comment]
Fixed.
